pkg/translator/jaeger/jaegerproto_to_traces.go

```diff
@@ -32,11 +32,17 @@ import (
 
 var blankJaegerProtoSpan = new(model.Span)
 
-// ProtoBatchesToInternalTraces converts multiple Jaeger proto batches to internal traces
+// Deprecated: [0.45.0] use `jaeger.ProtoToTraces`
 func ProtoBatchesToInternalTraces(batches []*model.Batch) pdata.Traces {
+	td, _ := ProtoToTraces(batches)
+	return td
+}
+
+// ProtoToTraces converts multiple Jaeger proto batches to internal traces
+func ProtoToTraces(batches []*model.Batch) (pdata.Traces, error) {
```

Comment:
Cleanup the name, as well as adding an error to allow possible errors to be return in the future.